sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2356,6 +2374,12 @@ class SQLConf extends Serializable with Logging {
   def partitionOverwriteMode: PartitionOverwriteMode.Value =
     PartitionOverwriteMode.withName(getConf(PARTITION_OVERWRITE_MODE))
 
+  def useStrictStoreAssignmentPolicy: Boolean =
```

Comment:
Instead of creating one boolean method for one mode, can we follow `partitionOverwriteMode` and only have one mode to return the enum?